sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/BooleanLiteralUnnecessary.vb

```diff
@@ -0,0 +1,194 @@
+﻿Imports System.Collections.Generic
+Imports System.Linq
+
+Public Class BooleanLiteralUnnecessary
+
+  Public Sub New(ByVal a As Boolean, ByVal b As Boolean, ByVal c As Boolean?)
+    Dim condition = False
+    Dim exp = True
+    Dim exp2 = True
+
+    Dim z = True OrElse ((True)) ' Noncompliant (also S1764)
```

Comment:
Could you also add a check with `And` and `Or` (the operators forcing both parts to be evaluated).

Reply:
really good catch, thanks!!!